docs(config): document index template priority configuration in 8.7 (#6469)
camunda/camunda-docs: versioned_docs/version-8.7/self-managed/operate-deployment/operate-configuration.md

schema is created, settings may be adjusted directly in the Elasticsearch template, and the new settings are applied
to indices created after adjustment.

#### Settings for index templates priority

Operate creates index templates that Elasticsearch uses for the historical indices. The priority of these templates can be changed.

This is useful when the Elasticsearch provider has some predefined wildcard (with `*` index pattern) index templates with given priority, setting a higher priority for Operate index templates ensures that the correct index mappings and settings are applied on the indices created from these templates.

The following configuration parameter defines the setting:

| Name                                                | Description                                         | Default value   |
| --------------------------------------------------- | --------------------------------------------------- | --------------- |
| camunda.operate.elasticsearch.indexTemplatePriority | Priority for all index templates created by Operate | - (no priority) |

This value is applied every time the configuration is changed.

:::note

> The priority should be different (strictly higher) from the wildcard index template one.

Reviewer: ```suggestion
The priority should be different (strictly higher) than that set by the wildcard templates.
```

schema is created, settings may be adjusted directly in the OpenSearch template, and the new settings are applied
to indices created after adjustment.

#### Settings for index templates priority

Operate creates index templates that OpenSearch uses for the historical indices. The priority of these templates can be changed.

This is useful when the OpenSearch provider has some predefined wildcard (with `*` index pattern) index templates with given priority, setting a higher priority for Operate index templates ensures that the correct index mappings and settings are applied on the indices created from these templates.

The following configuration parameter defines the setting:

| Name                                             | Description                                         | Default value   |
| ------------------------------------------------ | --------------------------------------------------- | --------------- |
| camunda.operate.opensearch.indexTemplatePriority | Priority for all index templates created by Operate | - (no priority) |

This value is applied every time the configuration is changed.

:::note

> The priority should be different (strictly higher) from the wildcard index template one.

Reviewer: ```suggestion
The priority should be different (strictly higher) than that set by the wildcard templates.
```
